### PR TITLE
Stripe form

### PR DESCRIPTION
**Pull request by stefanosu**
Created the payment UI component(s).

Created the backend communication to make a successful payment and customer object.

Created Redux action creators and actions related to payment and changing the global state to reflect the new state(s) from a successful payment

Need to add more styling to the payment card, payment success and pending pages.

And need to add unit tests as well.

### CODE REVIEW

**Review by dogpineapple — APPROVED** [review]
YEEEET

File: bloggies_frontend/src/RoutedPages/PaymentPage/index.tsx [hunk 1]
```diff
@@ -1,58 +1,114 @@
...
+  const elements = useElements();
+  const dispatch = useDispatch()
+  const userCustomerId = useSelector((st:CustomReduxState) => st.user.customer_id )
+  const history = useHistory()
+
+  const [error, setError] = useState(null);
+  const [cardComplete, setCardComplete] = useState(false);
+  const [processing, setProcessing] = useState(false);
+  const [paymentMethod, setPaymentMethod] = useState(null);
+  const [billingDetails, setBillingDetails] = useState({
```
Comment (megmcdevitt) [inline bloggies_frontend/src/RoutedPages/PaymentPage/index.tsx:27]: It doesn't look like we're using any of this state. Are you planning to do something with it later?

File: bloggies_backend/src/routes/stripe.ts [hunk 2]
```diff
@@ -75,6 +80,9 @@ stripeRouter.post("/create-checkout-session", async function (req: Request, res:
 stripeRouter.post("/create-customer", ensureLoggedIn, async function (req: Request, res: Response, next: NextFunction) {
   const { user_id, email } = req.user;
   const user = await User.getUser(user_id);
+  console.log(user_id) 
+  console.log(email) 
+  console.log(req.user)
```
Comment (megmcdevitt) [inline bloggies_backend/src/routes/stripe.ts:85]: No big deal, but we can probably remove these console logs, yeah?

**Review by megmcdevitt — APPROVED** [review]
Great job! Just a couple of small things. Check out some of the React warnings (it's mostly something was declared but never used). Approving anyway!

File: bloggies_backend/src/routes/stripe.ts (hunk 2 above)
Comment (stefanosu) [reply bloggies_backend/src/routes/stripe.ts:85]: Yes I'll remove that was logging to check some values.

File: bloggies_frontend/src/RoutedPages/PaymentPage/index.tsx (hunk 1 above)
Comment (stefanosu) [reply bloggies_frontend/src/RoutedPages/PaymentPage/index.tsx:27]: Yes was going to use that earlier but not anymore will remove it thanks.

**Pull request merged**